Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -48,18 +49,19 @@ public function configDataProvider(): array
                 [
                     'namespace' => 'myapp',
                     'type' => 'redis',
-                    'ignored_routes' => ['prometheus_bundle_prometheus'],
                     'redis' => [
                         'host' => '127.0.0.1',
                         'port' => 6379,
                         'timeout' => 0.1,
-                        'read_timeout' => 10,
+                        'read_timeout' => '10',
                         'persistent_connections' => false,
                         'password' => null,
                     ],
                     'disable_default_metrics' => false,
                     'disable_default_promphp_metrics' => false,
                     'enable_console_metrics' => false,
+                    'storage' => ['type' => 'redis'],
```

Comment:
The same comment here: why do we need to have a storage defined on a legacy use case? I'd assume that people who've just upgraded, shouldn't change their configuration.

Reply:
This change is on the expected part before compilation. We don't need to specify the storage. The configuration will inject values of `type` and `redis` to simplify usage in the extension that is why the storage appears in expected. The legacy config will still works fine.